includes/class-wc-gateway-stripe.php

```diff
@@ -859,11 +867,22 @@ public function change_no_available_methods_message() {
 	/**
 	 * Renders hidden inputs on the "Pay for Order" page in order to let Stripe handle PaymentIntents.
 	 *
+	 * @param WC_Order|null $order Order object, or null to get the order from the "order-pay" URL parameter
+	 *
 	 * @since 4.2
 	 */
-	public function render_payment_intent_inputs() {
-		$order     = wc_get_order( absint( get_query_var( 'order-pay' ) ) );
-		$intent    = $this->get_intent_from_order( $order );
+	public function render_payment_intent_inputs( $order = null ) {
```

Comment:
@dwainm Ideally, on the page where you want to show the SCA modal, you'll just need to:
- Include the extension's `stripe.js` script and the real `js.stripe.com/stripe.js` script (they're enqueued in certain pages by [this function](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/22b278392ff2cd7363d8ec1de1a1bd6f987f3275/includes/class-wc-gateway-stripe.php#L393))
- Use the `woocommerce_get_return_url` filter if you want to customize where to redirect the user after successful SCA authentication.
- Call this function.

The SCA modal will pop-up automatically.

Reply:
Thank you @DanReyLop!  I'll pull this into my test PR on Woo.com